### PR TITLE
Update image.ipynb / image annotation import - new paradigm

### PR DESCRIPTION
**Pull request by ovalle15**

### CODE REVIEW

File: examples/annotation_import/image.ipynb [hunk 1]
```diff
@@ -8,7 +8,7 @@
       },
```
Comment (msokoloff1) [inline examples/annotation_import/image.ipynb:8]: <p>Something is wrong with these links they aren't rendering properly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (msokoloff1) [inline examples/annotation_import/image.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #15.    import uuid
</pre><p>not really your change, but you might want to consider cleaning up imports. uuid is a duplicate.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (msokoloff1) [inline examples/annotation_import/image.ipynb:8]: <pre class="ql-syntax" spellcheck="false">Line #5.    mal_label_list = [li_label]
</pre><p>shouldn't this be called <code>li_label_list</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/744/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Comment (ovalle15) [reply examples/annotation_import/image.ipynb:8]: <p>The image renders properly in the colab notebook, is it possible this is an issue with app.reviewnb? </p>

Comment (ovalle15) [reply examples/annotation_import/image.ipynb:8]: <p>Sounds good </p>

Comment (ovalle15) [reply examples/annotation_import/image.ipynb:8]: <p>Yes, thank you for catching that !</p>

Comment (msokoloff1) [reply examples/annotation_import/image.ipynb:8]: yeah I think so, I realized this in a different review.

**Pull request merged**